docker/scripts/merge_env.py

```diff
@@ -10,10 +10,7 @@
 #
 import sys
 
-PREFERRED_BASE_ENV_KEYS = [
-    "STRUCTURE_TOOL_IMAGE_URL",
-    "STRUCTURE_TOOL_IMAGE_TAG",
-]
+PREFERRED_BASE_ENV_KEYS = []
```

Comment:
@chandrasekharan-zipstack This change needs to be reverted so that we always honor `sample.env` and use its contents to update `.env` each time.

Reply:
@hari-kuriakose we always update the `sample.env` with the latest structure tool version. However since we added these fields under `PREFERRED_BASE_ENV_KEYS` -> during the merge of envs, we don't override the value already present in the user's `.env`. As a result the user would always have an older version of structure tool present in their setup until and unless they manually check and update the value